platforms/software/testing-base/src/main/java/org/gradle/api/internal/tasks/testing/TestingBasePluginServices.java

```diff
@@ -21,10 +21,10 @@
 import org.gradle.internal.service.scopes.AbstractGradleModuleServices;
 
 public class TestingBasePluginServices extends AbstractGradleModuleServices {
-
     @Override
     public void registerBuildSessionServices(ServiceRegistration registration) {
         registration.addProvider(new TestExecutionBuildOperationBuildSessionScopeServices());
+        registration.addProvider(new TestServiceBuildSessionScopeServices());
```

Comment:
I think we should just have a single provider for "testing" build session services. 

The `TestExecutionBuildOperationBuildSessionScopeServices` only has a single service in it. I would vote to rename that and merge `TestServiceBuildSessionScopeServices` with it